src/main/webapp/app/home/home.scss

```diff
@@ -41,3 +41,13 @@ Main page styles
 .btn-space {
     margin-right: 5px;
 }
+
+@media (max-width: 768px) {
+    .col-md-8 {
+        width: 100% !important;
+    }
+
+    .col-md-6 {
+        width: 100% !important;
+    }
```

Comment:
Does adding col-sm-12 not work instead of doing this since !important is not the ideal solution?